components/UserEventsTable.js

```diff
@@ -1,22 +1,22 @@
 import React, { Component } from "react";
 import { StyleSheet, Text, View } from "react-native";
-// import { Row, Rows, Table } from "react-native-table-component";
+import { DataTable } from "react-native-paper";
 
 function UserEventsTable({ user }) {
-  const tableHead = ["Name", "Category", "Date", "Points"];
   let tableContents = [];
   if (user && user.events) {
     for (let i = 0; i < user.events.length; i++) {
       const event = user.events[i];
-      const date = event.createdAt.substring(5, 7) + "/"
-        + event.createdAt.substring(8, 10) + "/"
-        + event.createdAt.substring(0, 4);
+      const createdAt = new Date(event.createdAt);
+      const date = createdAt.getMonth() +"/" + createdAt.getDate() + "/" + createdAt.getFullYear();
 
       let row = [
-        event.name,
-        event.category,
-        date,
-        event.points
+        <DataTable.Row>
```

Comment:
Same as the other comments. It's an easy fix which is why I'm telling you to do it, I just don't like that the warning for that keeps appearing everywhere I go